deepfence_worker/worker.go

```diff
@@ -162,6 +165,14 @@ func NewWorker(ns directory.NamespaceID, cfg config) (Worker, context.CancelFunc
 		mux:       mux,
 		srv:       srv,
 		namespace: ns,
+		scan_workload_allocator: utils_ctl.NewRedisWorkloadAllocator(
+			cfg.MaxScanWorkload,
+			ns,
+			&directory.RedisConfig{
+				Endpoint: fmt.Sprintf("%s:%s", cfg.RedisHost, cfg.RedisPort),
+				Database: cfg.RedisDbNumber,
+				Password: cfg.RedisPassword},
+		),
```

Comment:
We have to get the Redis config from the context, otherwise it will be common to everyone?